[lldb-dap] Fix package.json after a bad merge.

ashgti opened the pull request:
The package.json is currently malformed after a bad merge in 39e6e888a8155583713e1b8b256119a2be7902e0.

Jlalond commented:
@ashgti Hey John, do you mind splitting this patch?

Because one part of it is fixing the bad merge, and one part is formatting. Putting them together makes the meat of this patch harder to follow.

As for the package.json, do we have any tests that try to npm build? Should we add them?

ashgti committed:
[lldb-dap] Fix package.json after a bad merge.

The package.json is currently malformed after a bad merge in 39e6e888a8155583713e1b8b256119a2be7902e0.

ashgti commented:
> @ashgti Hey John, do you mind splitting this patch?

Sure, I'll make this fixing the package.json.

> As for the package.json, do we have any tests that try to npm build? Should we add them?

I don't think they're run as part of the CI today. We could see if there is some way to integrate the npm build / tests into the lldb-dap CI somehow.

Jlalond reviewed (approved):
Thanks @ashgti!
